templates/base.html

```diff
@@ -28,8 +76,7 @@
     </div>
 
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
```

Comment:
To @tylor's point, I would be curious if we can remove jQuery as well. I suspect it can be removed since the only two JS libraries used in the project are `bootstrap==5.0.0` and `tinymce==4.7.4`, neither of which depend on jQuery. But we might need to do some additional checking to validate that, so probably best left for a small follow-up story. Also as an aside, I noticed that [tinymce](https://github.com/tinymce/tinymce) is now on v5.8.0.